ArcGISRuntimeSDKQt_QMLSamples/Maps/MinMaxScale/README.md

```diff
@@ -0,0 +1,19 @@
+# Min Max Scale
+
+This demonstrates how to set the minimum and maximum scale of an ArcGIS Map.
+
+![](screenshot.png)
+
+## How to use the sample
+Use the mousewheel to zoom in and out of the map. Notice that the scale of which you can do that is limited by the application.
+
+## How it works
+1. Create a MapView QML object, and create a Map child. Set the basemap and dimensions to whatever you like.
```

Comment:
Suggested rewording:

A MapView is declared with a nested Map object as a child. The streets basemap is then added as the map's basemap.